NaviGator/mission_control/navigator_launch/launch/vrx_trial.launch

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0"?>
```

Comment:
Will change with my upcomming rework of vrx launch system

Should be included by vrx.launch